### PR TITLE
hosted_engine_setup: Detect hosted-engine-ha version using /usr/libexec/platform-python

### PR DESCRIPTION
**Pull request by arachmani**
Bug-Url: https://bugzilla.redhat.com/2114769
Signed-off-by: Asaf Rachmani <arachman@redhat.com>

### CODE REVIEW

File: roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml [hunk 1]
```diff
@@ -110,9 +110,8 @@
     ignore_errors: true
     changed_when: true
   - name: Detect ovirt-hosted-engine-ha version
-    ansible.builtin.command: >-
-      {{ ansible_python.executable }} -c
-      'from ovirt_hosted_engine_ha.agent import constants as agentconst; print(agentconst.PACKAGE_VERSION)'
+    ansible.builtin.shell: >-
+      rpm -qi ovirt-hosted-engine-ha | grep "Version" | awk '{print $3}'
```
Comment (kobihk) [inline roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml:114]: maybe you can try this one:
```suggestion
      rpm -q --queryformat '%{VERSION}' ovirt-hosted-engine-ha
```

Comment (arachmani) [reply roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: Thanks, CI fails with the following error:
`command-instead-of-module rpm used in place of yum or rpm_key module` 
that comes from ansible-lint, so I'll try with the `yum` module.

**Review by mwperina — COMMENTED** [review]
Wouldn't it be more correct to use package facts to get a version of an installed package? For example:
https://github.com/oVirt/ovirt-engine/blob/master/packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/host-os.yml#L11
https://github.com/oVirt/ovirt-engine/blob/master/packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/host-os.yml#L31

**Comment by michalskrivanek** [comment]
or just use /usr/libexec/platform-python

**Comment by arachmani** [comment]
> Wouldn't it be more correct to use package facts to get a version of an installed package? For example: https://github.com/oVirt/ovirt-engine/blob/master/packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/host-os.yml#L11 https://github.com/oVirt/ovirt-engine/blob/master/packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/host-os.yml#L31

Yes, you are right, I already tried that and the only con is that the log is larger and contains irrelevant data for this task. I change it to use ` /usr/libexec/platform-python` as @michalskrivanek suggested

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**